From 6a88cc47b61656dc81fe527c2b9be91b125cc11e Mon Sep 17 00:00:00 2001
From: Pablo Santiago Blum de Aguiar <scorphus@gmail.com>
Date: Mon, 15 Jun 2015 09:20:07 -0300
Subject: [PATCH] =?UTF-8?q?fix(main.wait=5Foutput):=20use=20Process?=
 =?UTF-8?q?=E2=80=99=20children()=20instead=20of=20get=5Fchildren()?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Since psutil 2.0.0 `get_children()` has become deprecated and the use of
`children()` instead of it has been encouraged. In version 3.0.0, just
released, `get_children()` has been dropped. Check:

https://github.com/giampaolo/psutil/blob/master/HISTORY.rst

Fix #255
---
 thefuck/main.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thefuck/main.py b/thefuck/main.py
index 5772d4e..eebb668 100644
--- a/thefuck/main.py
+++ b/thefuck/main.py
@@ -62,7 +62,7 @@ def wait_output(settings, popen):
         proc.wait(settings.wait_command)
         return True
     except TimeoutExpired:
-        for child in proc.get_children(recursive=True):
+        for child in proc.children(recursive=True):
             child.kill()
         proc.kill()
         return False
